services/api-server/tests/unit/test_api_files.py

```diff
@@ -122,3 +133,63 @@ async def test_download_content(
 
     assert response.status_code == status.HTTP_200_OK
     assert response.headers["content-type"] == "application/octet-stream"
+
+
+async def test_get_upload_links(
```

Comment:
you could add some tests for checking the ValidationError issues:
such as ```test_get_upload_links_with_invalid_parameter_raises_422```
Normally I think that is how fastapi responds by default